### ISSUE TITLE
Plugin Admin deployment is broken

### ISSUE THREAD

**Issue by bviktor** [notepad-plus-plus/notepad-plus-plus#5035]
I'm trying to update our software bundle to include the new 7.6 release, but I'm facing many issues while migrating from the old plugin manager way.

1. While Notepad++ installs to `%ProgramFiles%\Notepad++`, the current "Plugin Admin" component installs as `%APPDATA%\Roaming\Notepad++\plugins\config\nppPluginList.dll`, which doesn't make any sense, since this is a system wide install, and a system wide component. So, essentially, the plugin admin will only be available for the user that installed Notepad++ and noone else. I tried copying around this DLL in the program folder, but to no avail, the Plugin Admin menu only shows up if it's under %APPDATA%.

2. Same goes for the other bundled plugins, DSpellCheck, mimeTools and NppConverter. Not available for other users, unless installing manually. A system wide installer shouldn't affect user preferences. If you want to make plugins strictly per-user, fine by me, but then take them out of the installer.

4. To add to the confusion, these bundled plugins are put into `%LOCALAPPDATA%\Notepad++\plugins` instead, which actually makes sense, since these are not configuration files. **But** then why is nppPluginList.dll in %APPDATA% instead?

5. The Plugin Admin also deploys 3 other files, `PluginManager.ini`, `PluginManagerPlugins.xml` and `PluginManagerPlugins.zip` into `%APPDATA%\Roaming\Notepad++\plugins\config`, and I have no idea what they're for, but plugin admin works without them.

6. Plugin Manager is missing from the portable version, which, again, doesn't make much sense to me, since normally you would want to carry over your plugins, especially since they're strictly per-user now.

7. Plugin Admin relies on GUP, which is also missing from the portable package.

---

My proposals:

1. Eliminate the bundled plugins from the installer, since they're 100% user files and preferences now. If you want to promote those 3 plugins, there are other means to do it, such as the plugin admin having a "featured plugins" tab, or a popup at first run, like "do you want to install these recommended plugins?", etc.

2. Install and handle Plugin Admin as a system wide file, such as `%ProgramFiles%\Notepad++\nppPluginList.dll`. This way it can work for all users.

3. Include both `nppPluginList.dll` and GUP (`updater\*`) in the zip/7z version too, so that people can use plugins while running in portable. Take into account that using the zip/7z version doesn't imply the user won't store her config under %APPDATA% / %LOCALAPPDATA%. It can be used merely because the user is not an administrator on the comp and thus doesn't have write permissions to %ProgramFiles%. All it takes is just deleting `doLocalConf.xml`. That's how we use it.

3+1. For the same reason, it'd be great if you also included the shell context menu DLL, `NppShell_06.dll` in the portable package. We're pushing Npp via robocopy, and the relevant registry settings via Group Policy, but since the shell DLL is missing from the zip package, we rely on an old DLL copied over from an older version.

**Comment by chcg** [comment]
See https://notepad-plus-plus.org/community/topic/16645/new-plugins-home-where-notepad-will-load-from, the discussion on this is already ongoing.
Also the lake of support from PA for the portable version is a known issue, see https://notepad-plus-plus.org/community/topic/16566/support-for-plugins-admin-npppluginlist/27

You have to distinguish between new https://notepad-plus-plus.org/news/notepad-7.6-released.html and longer available plugin https://github.com/bruderstein/nppPluginManager

**Comment by MetaChuh** [comment]
@bviktor 

sorry for the late reply.

your issues 1,2,3 and 6 have been fixed since 7.6.3, also including plugins admin for the portable versions.
issues 4 and 5 are not related to plugins admin, but to plugin manager.

your proposals 2,3 are done, plugin list is now in`%ProgramFiles%\Notepad++\plugins\config\nppPluginList.dll`
proposal 1 is an installer choice, which can be disabled at install time, and will not be changed for now, but plugins like dspellcheck have already been removed completely from the installer.

if you have not done it yet, please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins if you update from version 7.5.9 or below.
if your issue still persists, and it is still of relevance to you, please feel free to file a new issue as easy to understand and reproducible as possible.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)
